include/alpaka/dev/cpu/SysInfo.hpp

```diff
@@ -123,6 +123,26 @@ namespace alpaka
                 return std::string("unknown");
 #endif
             }
+
+            //! \return Pagesize in bytes used by the system.
+            inline size_t getPageSize()
+            {
+#if BOOST_OS_WINDOWS || BOOST_OS_CYGWIN
+                SYSTEM_INFO si;
+                GetSystemInfo(&si);
+                return si.dwPageSize;
+#elif BOOST_OS_UNIX || BOOST_OS_MACOS
+#    if defined(_SC_PAGESIZE)
+                return static_cast<std::size_t>(sysconf(_SC_PAGESIZE));
+#    else
+                return = static_cast<size_t>(getpagesize());
+#    endif
```

Comment:
Consider adding a comment that `getpagesize` is deprecated and that's why we have two versions here.